hibp.js

```diff
@@ -37,7 +37,12 @@ const HIBP = {
     } catch (error) {
       console.error(error);
     }
-    return foundBreaches;
+    for (const breach of foundBreaches) {
+      if ( breach.meta.IsVerified && !breach.meta.IsRetired && !breach.meta.IsSensitive && !breach.meta.IsSpamList ) {
+        filteredBreaches.push(breach);
+      }
+    }
+    return filteredBreaches;
```

Comment:
Silly comment, but not sure if it's easier to create a second array versus just doing something like this pseudo-shenanigans:

```js
return foundBreaches.filter(({meta}) => meta.IsVerified && !meta.IsRetired && !meta.IsSensitive && !meta.IsSpamList);
```

Reply:
Why silly? I was going to suggest this :)

Reply:
way better! not silly at all.